live-examples/js-examples/intl/intl-locale.html

```diff
@@ -0,0 +1,12 @@
+<pre>
+<code id="static-js">const korean = new Intl.Locale("ko", {script: "Kore", region: "KR", hourCycle: "h24", calendar: "gregory"});
+const japan = new Intl.Locale("ja-Jpan-JP-u-ca-japanese-hc-h12")
```

Comment:
Here and elsewhere, end the line with a semicolon.

Reply:
Also we should be consistent with variable names: so either:

"japan" and "korea"
or
"japanese" and "korean"